### ISSUE TITLE
Allow passing environment variables through

### ISSUE THREAD

**Issue by DanTup** [Microsoft/vscode-test#13]
Apologies if this exists, I can't find a way. Today when I run my tests I set a bunch of env variables in the CI script for things like log and coverage paths that I'll upload later:

```
env.DC_TEST_LOGS = path.join(cwd, ".dart_code_test_logs", `${testRunName}_${dartFriendlyName}_${codeFriendlyName}`);
env.COVERAGE_OUTPUT = path.join(cwd, ".nyc_output", `${testRunName}_${dartFriendlyName}_${codeFriendlyName}.json`);
env.TEST_XML_OUTPUT = path.join(cwd, ".test_results", `${testRunName}_${dartFriendlyName}_${codeFriendlyName}.xml`);
env.TEST_CSV_SUMMARY = path.join(cwd, ".test_results", `${dartFriendlyName}_${codeFriendlyName}_summary.csv`);
```

I can't see any obvious way to do this, if it's not supported would you consider it?

**Comment by octref** [comment]
`testRunnerEnv` is available in 0.4.0 now.

**Comment by disposedtrolley** [comment]
It doesn't look like this is working properly in `1.2.3`. Despite passing in the entirety of `process.env` to `runTests()`, no environment variables are available in the test runner script specified by `extensionTestsPath`:

```
await runTests({
  extensionDevelopmentPath,
  extensionTestsPath,
  extensionTestsEnv: process.env,
  launchArgs: [extensionWorkspacePath, "--disable-extensions"]
});
```

In the `extensionTestsPath` file:

```
export function run(): Promise<void> {
	console.log(process.env)  // prints nothing
}
```
